modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/IgnitePlanner.java

```diff
@@ -490,7 +491,7 @@ public void setDisabledRules(Set<String> disabledRuleNames) {
             List<RelOptRule> newSet = new ArrayList<>();
 
             for (RelOptRule r : rulesSet) {
-                if (!disabledRuleNames.contains(shortRuleName(r.toString()))) {
+                if (!disabledRuleNames.contains(shortRuleName(r))) {
```

Comment:
I would ask you to add real javadoc to any public methods which touched during modifications